hana/package.json

```diff
@@ -53,6 +53,13 @@
         }
       },
       "db": "sql"
+    },
+    "hana": {
+      "table_data": {
+        "column_mapping": {
+          "LargeBinary": "decodeBase64"
+        }
+      }
```

Comment:
Should that be default?

Reply:
It is, that's why I added it here.  Can be disabled with `LargeBinary: false`.
Other types can be registered, too.